auth_test.go

```diff
@@ -174,6 +175,26 @@ func TestAuthConfig(t *testing.T) {
 	}
 }
 
+func TestAuthConfigIdentityToken(t *testing.T) {
+	t.Parallel()
+	auth := base64.StdEncoding.EncodeToString([]byte("someuser"))
+	read := strings.NewReader(fmt.Sprintf(`{"auths":{"docker.io":{"auth":"%s","identitytoken":"sometoken"}}}`, auth))
+	ac, err := NewAuthConfigurations(read)
+	if err != nil {
+		t.Error(err)
```

Comment:
Can you replace this with ``t.Fatal(err)`` so the test execution aborts after failing?